soc/SoC_OCV.py

```diff
@@ -35,47 +34,70 @@ def __init__(self):
 		self.soc = [x for (x, y) in self.points]
 		self.v_ocv =  [y for (x, y) in self.points]
 		# Graph of the error of the SOC over the entire discharge (% error)
-		# Create BatTestPlotLinear.py (for more accuracy)
 		
 	def get_soc_and_v_ocv(self):
 		return self.soc, self.v_ocv
+	
+	def get_points(self):
+		return self.points
+	
+	def get_cell_ocv(self, soc: float or int):
+		if (soc > self.soc[-1] or soc < self.soc[0]):
+			raise Exception("The soc value if out of predicted range")
+		
+		max_soc = len(self.soc) - 1
+		min_soc = 0
 
-	# 
-	def get_cell_ocv(self, soc2):
-		if (soc2 > self.soc[-1] or soc2 < self.soc[0]):
-			return "Value is out of predicted range", "Value is out of predicted range"
-		v_ocv1 = 0
-		v_ocv2 = 0
+		while (min_soc <= max_soc):
```

Comment:
Don't need the brackets in `while` and `if` statements

Reply:
Ah, sorry it seems to be a habit from Java. I'll change that.